log/tracing/localStorage.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const { createNamespace } = require('continuation-local-storage')
+
+const INITIAL_REQUEST = 'initialRequest'
+
+const clsNamespace = createNamespace('uk.gov.hmcts.tracing.localStorage')
```

Comment:
maybe under the `reform` namespace?

Reply:
Yeah I wasn't sure about that. E.g. in Claim Store we don't reference Reform in out Java package structure.

Reply:
That's why I omitted it here as well.

Reply:
we usually put `reform` in for common components and libraries that have no true owner and are shared by everyone.